smart_open/s3.py

```diff
@@ -412,6 +412,26 @@ def truncate(self, size=None):
         """Unsupported."""
         raise io.UnsupportedOperation
 
+    def __str__(self):
+        return ("smart_open.s3.SeekableBufferedInputBase(%r, %r)") % (
```

Comment:
First set of parens not necessary here.

```suggestion
        return "smart_open.s3.SeekableBufferedInputBase(%r, %r)" % (
```